simplecache-core/build.sbt

```diff
@@ -14,4 +14,4 @@ libraryDependencies ++= Seq(
   "org.mockito" % "mockito-all" % "1.9.5" % "test",
```

Comment:
The final published POMs for this library do include e.g. junit, but seem to have these correctly marked as test scope. What do you think?

        <dependency>
            <groupId>junit</groupId>
            <artifactId>junit</artifactId>
            <version>4.4</version>
            <scope>test</scope>
        </dependency>